src/rgw/rgw_sal_motr.cc

```diff
@@ -2568,7 +2617,7 @@ int MotrAtomicWriter::complete(size_t accounted_size, const std::string& etag,
   ent.meta.owner = owner.to_str();
   ent.meta.owner_display_name = obj.get_bucket()->get_owner()->get_display_name();
   RGWBucketInfo &info = obj.get_bucket()->get_info();
-  if (info.versioning_enabled())
+  if (info.versioned())
```

Comment:
so whats the difference between versioning_enabled & versioned, why we need this change ?